com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/shader/ShaderCompilePipeline.java

```diff
@@ -92,6 +92,7 @@ private static int shaderLanguageToVersion(ShaderDesc.Language shaderLanguage) {
             case LANGUAGE_GLES_SM300 -> 300;
             case LANGUAGE_GLSL_SM330 -> 330;
             case LANGUAGE_GLSL_SM430 -> 430;
+            case LANGUAGE_HLSL       -> 50;
```

Comment:
I don't know if we need to support other versions here, but we'll add that later in that case. This is what one of our consoles requires currently